docs/source/use_CA.md

```diff
@@ -0,0 +1,205 @@
+
+### Creating the local MSP of a node
+
+While the MSP of an organization serves as the representation of the organization on a channel configuration, the local MSP of a node is a logical collection of parameters that is used, along with other parameters, as part of the creation of a node.
+
+As has been noted above, nodes must be bootstrapped both with enrollment certificates (the public and private key pair that identifies a node) and the TLS certificates that encrypt the communication layer between nodes. This "bootstrapping" happens by listing the location of these certificates in the relevant YAML configuration file that is referenced when creating the node.
+
+For reference, here is a sample [peer configuration file](https://github.com/hyperledger/fabric/blob/release-2.1/sampleconfig/core.yaml). And here is a sample [ordering node configuration file](https://github.com/hyperledger/fabric/blob/release-2.1/sampleconfig/orderer.yaml).
+
+Note that these configuration files ask for the location of the relevant local MSP folder. For the peer, this is defined through the `mspConfigPath`. For the orderer, it is the `LocalMSPDir`. The folders found in this location will be used to define the local MSP of the node, including the private key the node will use when signing its actions as well as the public key of at least one admin of the node.
+
+The TLS certificates, on the other hand, are defined individually, rather than pointing to a folder, and can be found in the `TLS settings` section of the YAML. This means that TLS certificates do not need to be kept in a strict folder structure like the local MSP (relevant in particular to users who will be using an external CA to generate TLS certificates --- use the sample YAML files as a guide to what these certificates are used for).
+
+As with all of the configuration parameters in the YAML file of a node, you have the option to specify the `msp` folder and TLS certificate locations either in the YAML itself or through the use of environment variables.
+
+Note: **it is a best practice to mount these folders (volumes) external to the container where the node itself is running. This will allow the certificates to be used to create a new node should the node container go down become corrupted.**
+
+For a look at a sample local MSP, check out [MSP structure](https://hyperledger-fabric.readthedocs.io/en/master/membership/membership.html#msp-structure). Note that you will not receive all of these certificates back simply by enrolling a peer identity. You will need, for example, to create the `users` subfolder and put the public certificate of the identity that will be administering the node in the folder prior to bootstrapping. You will also need an operations certificate (depending on the configuration of your network, this might come from a separate operations CA).
```

Comment:
You will need, for example, to create the `users` subfolder and put the public certificate of the identity that will be administering the node in the folder prior to bootstrapping.

Ah! I never knew that. I always wondered what that was for! Thank you.

But I am not sure what you mean by an "operations" certificate. I think we need to explain that one a little more since it has not been mentioned before.

Reply:
I was desperately trying to avoid a discussion of the operations service here. I'll add a link.